File: Codes/Dalily_Flash/Week_3/28Jan/Java/Program5.java
import java.util.*;

class MinNum {

	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);
		System.out.print("Enter No1 = ");
		int no1 = sc.nextInt();
		System.out.print("Enter No2 = ");
		int no2 = sc.nextInt();
		System.out.print("Enter NO3 = ");
		int no3 = sc.nextInt();
		if(no1<no2 && no1<no3) {

			System.out.println("The Minimun no amongst "+no1+" "+no2+" & "+no3+" is "+no1);
		} else if(no2<no3 && no2<no1) {
			System.out.println("The Minimun no amongst "+no1+" "+no2+" & "+no3+" is "+no2);
		} else {
			System.out.println("The Minimun no amongst "+no1+no2+" "+" & "+no3+" is "+no3);
	
		}
	}
}

File: Codes/Dalily_Flash/Week_3/28Jan/Python/Program5.py
no1 = int(input("Enter No1 = "))
no2 = int(input("Enter No2 = "))
no3 = int(input("Enter No3 = "))
if(no1<no2 and no1<no3):
    print("The min no. amongst",no1,no2,"and",no3,"is",no1)
elif(no2<no1 and no2<no3):
    print("The min no. amongst",no1,no2,"and",no3,"is",no2)
else:
    print("The min no. amongst",no1,no2,"and",no3,"is",no3)
